marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/entities/search/impl/CreatedOnFacetHandlerTest.java

```diff
@@ -4,38 +4,171 @@
 import com.marklogic.client.query.StructuredQueryDefinition;
 import com.marklogic.hub.central.entities.search.Constants;
 import com.marklogic.hub.central.entities.search.models.DocSearchQueryInfo;
+import com.marklogic.hub.central.exceptions.DataHubException;
 import com.marklogic.rest.util.Fragment;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class CreatedOnFacetHandlerTest extends AbstractFacetHandlerTest {
 
+    List<String> stringValues;
+    Map<String, String> dateRange;
+
+    CreatedOnFacetHandler createdOnFacetHandler;
```

Comment:
Storing these as class fields is fine because we don't have to worry about multiple threads running this same test instance.